apps/web/src/components/site-header.tsx

```diff
@@ -11,6 +12,8 @@ import { UserMenu } from "./user-menu";
 
 export async function SiteHeader() {
   const user = await getUser();
+  const programmerbarStatus = (await getProgrammerbarStatus()).message;
+  const sucsessfullCall = (await getProgrammerbarStatus()).success;
```

Comment:
```suggestion
  const { success, message } = await getProgrammerbarStatus()
```

Her kan det være lurt å bare kjøre funksjonen 1 gang, hvis ikke gjør vi to requests til det endepunktet. Dette gjør at det tar dobbelt så langt til å rendre komponenten da (er snakk om millisekunder da). Men det kan være lurt å bare bruke den samme funksjonen begge gangene.